menu/navigation.json

```diff
@@ -2312,6 +2312,74 @@
             ],
             "label": "Redis™",
             "slug": "redis"
+          },
+          {
+            "items": [
+              {
+                "label": "MongoDB™ Overview",
```

Comment:
If it's Private Beta I feel like we normally wouldn't include it in the menu? (until Public Beta)

Reply:
@scw-wtimmermans do we keep it in the menu?

Reply:
that is correct, not in the menu until Public Beta. What is the best way for users to access it in that case?